plugins/GeoIp2/GeoIp2.php

```diff
@@ -50,4 +58,54 @@ public function getClientSideTranslationKeys(&$translationKeys)
         $translationKeys[] = "General_Save";
         $translationKeys[] = "General_Continue";
     }
+
+    /**
+     * Customize the Installation "default settings" form.
+     *
+     * @param FormDefaultSettings $form
+     */
+    public function installationFormInit(FormDefaultSettings $form)
+    {
+        $form->addElement('checkbox', 'setup_geoip2', null,
+            [
+                'content' => '<div class="form-help">' . Piwik::translate('GeoIp2_AutomaticSetupDescription', ['<a rel="noreferrer noopener" href="https://db-ip.com/?refid=mtm">','</a>']) . '</div> &nbsp;&nbsp;' . Piwik::translate('GeoIp2_AutomaticSetup')
+            ]
+        );
+
+        // default values
+        $form->addDataSource(new \HTML_QuickForm2_DataSource_Array([
+            'setup_geoip2' => true,
+        ]));
+    }
+
+    /**
+     * Process the submit on the Installation "default settings" form.
+     *
+     * @param FormDefaultSettings $form
+     */
+    public function installationFormSubmit(FormDefaultSettings $form)
+    {
+        $setupGeoIp2 = (bool) $form->getSubmitValue('setup_geoip2');
+
+        if ($setupGeoIp2) {
+            Option::set(GeoIP2AutoUpdater::AUTO_SETUP_OPTION_NAME, true);
+            GeoIP2AutoUpdater::setUpdaterOptions([
+                'loc' => \Piwik\Plugins\GeoIp2\LocationProvider\GeoIp2::getDbIpLiteUrl(),
+                'period' => GeoIP2AutoUpdater::SCHEDULE_PERIOD_MONTHLY
+            ]);
+
+            /** @var Scheduler $scheduler */
+            $scheduler = StaticContainer::getContainer()->get('Piwik\Scheduler\Scheduler');
+            $scheduler->rescheduleTask(new GeoIP2AutoUpdater());
+
+            $cliMulti = new CliMulti();
+
+            if ($cliMulti->supportsAsync()) {
+                $phpCli = new CliMulti\CliPhp();
+                $command = sprintf('%s %s/console core:run-scheduled-tasks > %s/tmp/out.txt 2>&1 &',
```

Comment:
@sgiehl might be good to run eg `console core:run-scheduled-tasks  --force "Piwik\Plugins\GeoIp2\GeoIP2AutoUpdater.update"` directly? Also is the redirection needed to out.txt needed? Eg on nginx by default the file would maybe be directly accessible. Not a big issue but might just not be needed?

Reply:
the output shouldn't be needed, that was for debugging purpose 🙈 
will change that to directly execute the task.